src/docs/sphinx/tutorials/step11/Tutorial.rst

```diff
@@ -0,0 +1,234 @@
+   :width: 500
+   :figclass: align-center
+
+   Fig.2 Generated mesh for the wellbore problem
+
+
+Let us take a closer look at the geometry of this wellbore problem.
+We use the internal mesh generator ``InternalWellbore`` to create a rock domain
+(:math:`10\, m \, \times 5 \,  m \, \times 2 \, m`), with a wellbore of
+initial radius equal to :math:`0.1` m.
+Coordinates of ``trajectory`` defines the wellbore trajectory, which represents a perfect vertical well in this tutorial. 
+By turning on ``autoSpaceRadialElems="{ 1 }"``, the internal mesh generator automatically sets number and spacing of elements in the radial direction, which overrides the values of ``nr``. 
+With ``useCartesianOuterBoundary="0"``, a Cartesian aligned outer boundary on the outer block is enforced.
+In this way, a structured three-dimensional mesh is created with 100 x 80 x 2 elements in the radial, tangential and z directions, respectively. All the elements are eight-node hexahedral elements (``C3D8``) and refinement is performed
+to conform with the wellbore geometry. This mesh is defined as a cell block with the name
+``cb1``.
+
+
+.. literalinclude:: ../../../../../examples/plasticity/DruckerPrager_Verification.xml
+    :language: xml
+    :start-after: <!-- SPHINX_DP_WELLBORE_MESH -->
+    :end-before: <!-- SPHINX_DP_WELLBORE_MESH_END -->
+
+
+Solver: setting up the solid mechanics solver
+-----------------------------------------------------------
+
+For the drained wellbore problem, the pore pressure variation is omitted and can be subtracted from the analysis. Therefore, we just need to define a solid mechanics solver, which is called ``mechanicsSolver``. 
+This solid mechanics solver (see :ref:`SolidMechanicsLagrangianFEM`) is based on the small strain Lagrangian finite element formulation. 
+The problem is run as ``QuasiStatic`` without considering the inertial. 
+The computational domain is discretized by ``FE1``, which is defined in the ``NumericalMethods`` section. 
+The material is named as ``rock``, whose mechanical properties are specified in the ``Constitutive`` section.
+
+.. literalinclude:: ../../../../../examples/plasticity/DruckerPrager_Verification.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DP_WELLBORE_SOLVER -->
+  :end-before: <!-- SPHINX_DP_WELLBORE_SOLVER_END -->
+
+
+Constitutive model: defining material properties with constitutive laws
+-----------------------------------------------------------
+
+For this drained wellbore problem, we simulate the elastoplastic deformation caused by wellbore contraction.
+A homogeneous domain with one solid material is assumed, whose mechanical properties are specified in the ``Constitutive`` section: 
+
+.. literalinclude:: ../../../../../examples/plasticity/DruckerPrager_Verification.xml
+    :language: xml
+    :start-after: <!-- SPHINX_DP_WELLBORE_MATERIAL -->
+    :end-before: <!-- SPHINX_DP_WELLBORE_MATERIAL_END -->
+
+
+Recall that in the ``SolidMechanicsLagrangianSSLE`` section, 
```

Comment:
SSLE = small strain linear elastic.  It's probably not appropriate to use this solver tag for an elastoplastic problem.

Reply:
Right, it is not the right mechanical solver in this case and will change it